### ISSUE TITLE
Mathematical formulas visualisation problem

### ISSUE THREAD

**Issue by spapastamkou** [programminghistorian/jekyll#2053]
There is a problem of visualizing some mathematical formulas in the[ preview](http://programminghistorian.github.io/ph-submissions/fr/lecons/calibration-radiocarbone-avec-r) of a FR lesson currently under review (issue [#329](https://github.com/programminghistorian/ph-submissions/issues/329) in ```ph-submissions```. Some were resolved in the beginning (by replacing ^ with html sup tag for superscripts) but trouble persists with others. I open this issue bcz I will probably need advice from members who are able to provide some. 

![formule-math](https://user-images.githubusercontent.com/27706167/109083937-212ce980-7707-11eb-9048-dcb402a30a46.png)

**Comment by rivaquiroga** [comment]
@spapastamkou, if the lesson has formulas, you need to add the `mathjax: true` field to the YAML header.

**Comment by rivaquiroga** [comment]
Maybe we should add this to the Editor Guidelines. What do you think?

**Comment by spapastamkou** [comment]
That should be it and it is as simple as that, thank you @rivaquiroga !

**Comment by spapastamkou** [comment]
I will keep this ticket open until I make the pull request for the lesson (which should not take much longer). I still have the visualisation problem in ```ph-submissions``` even with `mathjax: true` in YAML and I'll wait to see what will happen once I move the lesson in `jekyll`.

**Comment by rivaquiroga** [comment]
@spapastamkou, in `ph-submissions` you should be able to see them correctly.
I think the problem might be the delimeters that the author is using. For in-line formulas (like the ones from the screenshot) you should use `\(...\)`. When they are not in-line, you can use `$$...$$` or `\[...\]`.

**Comment by spapastamkou** [comment]
@rivaquiroga Thanks Riva, I'll try this suggestion as I check the syntax of the lesson (I had tested current delimeters in a LaTex editor and there it worked fine).

**Comment by rivaquiroga** [comment]
@spapastamkou , yes, on a LaTex editor `$...$` should work fine. But Mathjax don't use it because the dollar sign appears too often in other contexts (like when writing amounts of money). 

Maybe we should explain what delimeters are allowed in the Author Guidelines.

**Comment by spapastamkou** [comment]
@rivaquiroga We may need to consider documenting the wiki, as this can be tricky. I put here an html block for which it took me some time to find the solution. In case math formulas are in a html block, must also insert a **div with a class mathjax**: 
```
<div class="alert alert-warning">
	<div class="mathjax">
  <p>Pour convertir en années BC/AD un âge <i>calibré</i> (noté \(x\)) exprimé en années BP, sachant qu'il n'y pas d'année 0 dans le calendrier grégorien :
  <ul>
  <li>Si l'âge calibré est inférieur à 1950 BP : \(1950 - x\) AD</li>
  <li>Si l'âge calibré est supérieur ou égal à 1950 BP : \(1949 - x\) BC</li>
  </ul>
  </p> 
  	</div>
</div>
```

**Comment by spapastamkou** [comment]
Problem resolved, so I sum up: 
- for inline formulas, what worked here was: `\\(...\\)` (double antislash)
- for centered formulas, `$$...$$` worked fine
- in a html block, I had to create a `<div class="mathjax">`
Knowing that, I had checked @acrymble 's lesson on gravity models and there inline formulas with (unique) dollars worked fine. 

This ticket will be closed with PR #2077
